src/olympia/addons/admin.py

```diff
@@ -649,6 +649,22 @@ def save_model(self, request, obj, form, change):
         self._send_mail(obj, 'changed' if change else 'added')
 
 
+@admin.register(models.AddonBrowserMapping)
+class AddonBrowserMappingAdmin(AMOModelAdmin):
+    list_display = ('addon__name', 'browser', 'extension_id', 'created', 'modified')
+    fields = ('addon', 'browser', 'extension_id')
+    raw_id_fields = ('addon',)
+    readonly_fields = ('created', 'modified')
+
+    def get_readonly_fields(self, request, obj=None):
+        return self.readonly_fields + (('addon',) if obj else ())
```

Comment:
Do we really care ? I understand this is not a change to be done lightly but ff there is a mistake it'd be nice to be able to correct it without having to delete and re-create the entry.